modules/data.atmosphere/R/pecan_standard_met_table.R

```diff
@@ -0,0 +1,24 @@
+  ~`cf_standard_name`                                     , ~units        , ~bety               , ~isimip        , ~cruncep , ~narr   , ~ameriflux         ,
+  "air_temperature"                                       , "K"           , "airT"              , "tasAdjust"    , "tair"   , "air"   , "TA (C)"           ,
+  "air_temperature_max"                                   , "K"           , NA                  , "tasmaxAdjust" , NA       , "tmax"  , NA                 ,
+  "air_temperature_min"                                   , "K"           , NA                  , "tasminAdjust" , NA       , "tmin"  , NA                 ,
+  "air_pressure"                                          , "Pa"          , "air_pressure"      , NA             , NA       , NA      , "PRESS (KPa)"      ,
+  "mole_fraction_of_carbon_dioxide_in_air"                , "mol/mol"     , NA                  , NA             , NA       , NA      , "CO2"              ,
+  "moisture_content_of_soil_layer"                        , "kg m-2"      , NA                  , NA             , NA       , NA      , NA                 ,
+  "soil_temperature"                                      , "K"           , "soilT"             , NA             , NA       , NA      , "TS1 *(NOT DONE)*" ,
+  "relative_humidity"                                     , "%"           , "relative_humidity" , "rhurs"        , NA       , "rhum"  , "RH"               ,
+  "specific_humidity"                                     , "1"           , "specific_humidity" , NA             , "qair"   , "shum"  , "CALC(RH)"         ,
+  "water_vapor_saturation_deficit"                        , "Pa"          , "VPD"               , NA             , NA       , NA      , "VPD *(NOT DONE)*" ,
+  "surface_downwelling_longwave_flux_in_air"              , "W m-2"       , "same"              , "rldsAdjust"   , "lwdown" , "dlwrf" , "Rgl"              ,
+  "surface_downwelling_shortwave_flux_in_air"             , "W m-2"       , "solar_radiation"   , "rsdsAdjust"   , "swdown" , "dswrf" , "Rg"               ,
+  "surface_downwelling_photosynthetic_photon_flux_in_air" , "mol m-2 s-1" , "PAR"               , NA             , NA       , NA      , "PAR *(NOT DONE)*" ,
+  "precipitation_flux"                                    , "kg m-2 s-1"  , "cccc"              , "prAdjust"     , "rain"   , "acpc"  , "PREC (mm/s)"      ,
+  "wind_to_direction"                                     , "degrees"     , "wind_direction"    , NA             , NA       , NA      , "WD"               ,
+  "wind_speed"                                            , "m/s"         , "Wspd"              , NA             , NA       , NA      , "WS"               ,
+  "eastward_wind"                                         , "m/s"         , "eastward_wind"     , NA             , NA       , NA      , "CALC(WS+WD)"      ,
+  "northward_wind"                                        , "m/s"         , "northward_wind"    , NA             , NA       , NA      , "CALC(WS+WD)"
+)
```

Comment:
Is the intention to export this as a dataset? If so, I'd handle it something like PEcAn.utils::standard_vars, i.e. create the object in `data/pecan_standard_met_table.R`, document it in something like `R/datasets/.R`. If it's not intended to be user-visible and is strictly for internal use, I think Hadley recommends `R/sysdata.rda`

Reply:
So this is designed to be both useful internally and user-facing. As for how exactly it's stored, I'm not sure I see the benefit of storing it in `data/` or `systdata.rda`. Things in `data` either have to be in `.Rdata` format -- which requires them to be compiled elsewhere, and therefore adds another step to installing this package whenever that changes -- or CSV -- which admittedly is basically what this is already...but this way, the dataset documentation and raw data live in the same location. 

In general, I was following [this recommendation](http://r-pkgs.had.co.nz/data.html) from the "R Packages":

> A simple alternative to these three options is to include it in the source of your package, either creating by hand, or using dput() to serialise an existing data set into R code.

...which I took to mean the way this is presented here.